sqldef.go

```diff
@@ -20,7 +20,7 @@ type Options struct {
 
 // Main function shared by `mysqldef` and `psqldef`
 func Run(generatorMode schema.GeneratorMode, db adapter.Database, options *Options) {
-	currentDDLs, err := adapter.DumpDDLs(db)
+	currentDDLs, constraints, err := adapter.DumpDDLs(db)
```

Comment:
The current design is to compare bare string DDLs and produce DDLs, instead of returning well-structured data from `DumpDDLs`. 

Could you please try not to change existing APIs used by other adapters? At least in this place, I thought you could produce DDLs for the constraints and let the parser parse them. That allows us to symmetrically implement the parsed results and make it harder to introduce inconsistency on one side.

Reply:
Thanks for the quick reply.
> At least in this place, I thought you could produce DDLs for the constraints and let the parser parse them.

It certainly looks like I can include existing constraint names in the DDLs.

I'll try to come up with an implementation that doesn't change the current structure.